topics/data-science/tutorials/online-resources-gene/tutorial.md

```diff
@@ -92,76 +175,197 @@ This panel reports the DNA sequence of the introns (in green), as well as the on
 > > <solution-title></solution-title>
 > >
 > > 1. The correspondent protein starts with Methionine, M (they all do).
-> > 2. It ends with the last AA of the last intron, hence Alanine, A. After that, the stop codon TGA.
+> > 2. It ends with the last AA of the last intron (found in the 2nd page), hence Alanine, A. After that, the stop codon TGA.
```

Comment:
**The last AA of the last exon (found in the 2nd page) is Alanine (A). After that, the stop codon TGA comes, which is not translated into an AA.**